### PR TITLE
Internet Radio Release 2.0.0.0

### PR DESCRIPTION
**Pull request by bethoma**
Notes:
	- Reafctor logic/management into sepearte Windows Runtime
	  Component
	- Add headed application for Device Center use

### CODE REVIEW

**Comment by mwmckee** [comment]
add "InternetRadioDevice/InternetRadio/project.lock.json" to .gitignore. Better to not put in ~14k lines of auto-generated code :)

File: InternetRadioDevice/InternetRadioHeaded/InternetRadioHeaded.csproj [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
+    <TargetPlatformVersion>10.0.10240.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.10240.0</TargetPlatformMinVersion>
+    <MinimumVisualStudioVersion>14</MinimumVisualStudioVersion>
+    <EnableDotNetNativeCompatibleProfile>true</EnableDotNetNativeCompatibleProfile>
+    <FileAlignment>512</FileAlignment>
+    <ProjectTypeGuids>{A5A43C5B-DE2A-4C0C-9213-0A381AF9435A};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+    <PackageCertificateKeyFile>InternetRadioHeaded_TemporaryKey.pfx</PackageCertificateKeyFile>
+    <PackageCertificateThumbprint>FE24E2AF9966F3EFA05F22B1ABC18D832B617EDD</PackageCertificateThumbprint>
```
Comment (mwmckee) [inline InternetRadioDevice/InternetRadioHeaded/InternetRadioHeaded.csproj]: If the .pfx file isn't being checked in, lines 20 and 21 should be removed (The "PackageCertificateKeyFile" and "PackageCertificateThumbprint" tags), so the project can build without warnings/errors.

**Comment by mwmckee** [comment]
signoff

File: InternetRadioDevice/InternetRadio/InternetRadio.csproj [hunk 2]
```diff
@@ -14,130 +14,167 @@
...
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x86'">
+    <PlatformTarget>x86</PlatformTarget>
+    <DebugSymbols>true</DebugSymbols>
+    <OutputPath>bin\x86\Debug\</OutputPath>
     <DefineConstants>DEBUG;TRACE;NETFX_CORE;WINDOWS_UWP</DefineConstants>
     <NoWarn>;2008</NoWarn>
     <DebugType>full</DebugType>
-    <PlatformTarget>ARM</PlatformTarget>
+    <PlatformTarget>x86</PlatformTarget>
```
Comment (mwmckee) [inline InternetRadioDevice/InternetRadio/InternetRadio.csproj]: I believe this was a file that was moved around when Ben created new projects, and GitHub is confused, so it just re-ordered all of the targets listed. It looks like all of the targets match their parent tag (x86\debug bin placing is under the "Debug|x86" tag, etc), and that all platforms are present.

**Comment by HerrickSpencer** [comment]
Signed off, ship it.

**Pull request merged**